lib/msal-common/src/account/AccountInfo.ts

```diff
@@ -17,4 +17,5 @@ export type AccountInfo = {
     tenantId: string;
     username: string;
     name?: string;
+    localAccountId?: string;
```

Comment:
While OID is optional claim (requires `profile`), I think that SUB (subject) is always present in ID tokens.
So this field could be safely not nullable? Or I miss something?

Reply:
Yes. Except for `B2C` custom policies where there can be some exceptions. I am checking the data with B2C service team and will close this soon.

Reply:
We decided to set it to "" in case of no `sub` use case, which shouldn't occur in any of the usual scenarios.